xcm/v6/transfers.json

```diff
@@ -424,6 +424,30 @@
         "sizeFactor": "3333333",
         "alwaysHoldingPays": false
       }
+    },
+    "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3": {
+      "toParachain": {
+        "type": "exponential",
+        "factorPallet": "Dmp",
+        "sizeBase": "300000000",
+        "sizeFactor": "1000000"
+      }
+    },
+    "68d56f15f85d3136970ec16946040bc1752654e906147f7e43e9d539d7c3de2f": {
+      "toParent": {
+        "type": "exponential",
+        "factorPallet": "ParachainSystem",
+        "sizeBase": "300000000",
+        "sizeFactor": "1000000",
```

Comment:
Must be 100_000. https://github.com/polkadot-fellows/runtimes/blob/v1.1.2/system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs#L249C42-L249C52